modules/popup/css/lib/popup.scss

```diff
@@ -1,33 +1,37 @@
-.wdc-popup {
+@mixin wdc-popup-common() {
   @include wdc-type();
   @include card-common();
   position: relative;
   animation-duration: 150ms;
   margin: 0;
   animation-timing-function: ease-out;
 
-  .wdc-popup-close {
+  @include popupAnimationOrigins();
+
+  &-padding-s {
+    padding: $wdc-spacing-s;
+  }
+
+  &-no-padding {
+    padding: 0;
+  }
+
+  &-close {
     position: absolute;
     right: $wdc-spacing-xs;
     top: $wdc-spacing-xs;
   }
 
-  .wdc-popup-title {
+  &-title {
```

Comment:
Rather than define separate `wdc-popup-title` and `wdc-popup-body` mixins, I used the ampersand within the `wdc-popup-common` mixin.